# Generated by Django 4.0.3 on 2022-04-21 15:34

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Item',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('isBin', models.BooleanField()),
                ('isVerified', models.BooleanField(default=False)),
                ('isAccepted', models.BooleanField(null=True)),
                ('latitude', models.IntegerField()),
                ('longitude', models.IntegerField()),
                ('image', models.ImageField(default='defaults.jpg', upload_to='default/', verbose_name='Image')),
                ('validatorVerdict1', models.BooleanField(null=True)),
                ('validatorVerdict2', models.BooleanField(null=True)),
                ('validatorVerdict3', models.BooleanField(null=True)),
                ('owner', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
                ('validator1', models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='validator1', to=settings.AUTH_USER_MODEL)),
                ('validator2', models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='validator2', to=settings.AUTH_USER_MODEL)),
                ('validator3', models.ForeignKey(null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='validator3', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
